README.rst

```diff
@@ -18,9 +18,9 @@ more information, see this `blog post <https://aws.amazon.com/blogs/developer/an
 
 .. _boto: https://docs.pythonboto.org/
 .. _`doc site`: https://boto3.amazonaws.com/v1/documentation/api/latest/index.html
-.. |Build Status| image:: http://img.shields.io/travis/boto/boto3/develop.svg?style=flat
-    :target: https://travis-ci.org/boto/boto3
-    :alt: Build Status
+.. |Run tests| image:: https://github.com/boto/boto3/actions/workflows/run-tests.ybadge.svg?branch=develop
```

Comment:
I think this'll actually break the badge. It's being inserted into the doc above as `Build Status` and this definition no longer matches. We can probably keep it as build status. It would be nice if we could change the label on the badge, but I don't see a way to do that through the API Github provides.

Reply:
Fixed, thanks!